Automattic/pocket-casts-ios
# Update Profile's header

This PR updates the Profile header so it's cleaner.

<img src="https://user-images.githubusercontent.com/7040243/235235780-544176a9-0325-471c-b947-cf3e382830de.png" width="300">


## To test

1. Run the app
2. Make sure you're logged out
3. ✅ Check the profile header
4. Login
5. ✅ Check the profile header
6. Set to Plus
7. ✅ The profile header should reflect you have Plus
8. Set to expiring plus
9. ✅ You should see a message where the email was appearing

Also, try the new header with different themes and accessibility fonts. For accessibility, you'll have to re-run the app. Also note that the "Account" button is not accessible yet, since it's used in other places and I thought this is out of scope for this task.

## Checklist
